tests/performance/Makefile

```diff
@@ -30,7 +30,13 @@ __check_defined = \
 
 .PHONY: build
 build: ## builds distributed osparc locust docker image
-	docker buildx build --load --build-arg LOCUST_VERSION=$(LOCUST_VERSION) --tag itisfoundation/locust:$(LOCUST_VERSION) --tag local/locust:latest .
+	docker --debug \
+		buildx build \
+		--load  \
+		--build-arg LOCUST_VERSION=$(LOCUST_VERSION) \
+		--tag itisfoundation/locust:$(LOCUST_VERSION) \
```

Comment:
what will happen with your default version in the dockerfile here?

Reply:
If there is no default, I get the following
```
 1 warning found (use docker --debug to expand):
 - InvalidDefaultArgInFrom: Default value for ARG locustio/locust:${LOCUST_VERSION} results in empty or invalid base image name (line 3)
``` 
adding `--debug`, it logs
```
1 warning found:
 - InvalidDefaultArgInFrom: Default value for ARG locustio/locust:${LOCUST_VERSION} results in empty or invalid base image name (line 3)
Default value for global ARG results in an empty or invalid base image name
More info: https://docs.docker.com/go/dockerfile/rule/invalid-default-arg-in-from/
Dockerfile:3
--------------------
   1 |     
   2 |     ARG LOCUST_VERSION
   3 | >>> FROM locustio/locust:${LOCUST_VERSION}
   4 |     
   5 |     
--------------------
```
in the link https://docs.docker.com/reference/build-checks/invalid-default-arg-in-from/, they suggest to add a default.

I will rather default to `latest` then.